pkg/kubelet/cm/cgroup_manager_linux.go

```diff
@@ -395,15 +394,6 @@ func getSupportedUnifiedControllers() sets.String {
 
 func (m *cgroupManagerImpl) toResources(resourceConfig *ResourceConfig) *libcontainerconfigs.Resources {
 	resources := &libcontainerconfigs.Resources{
-		Devices: []*libcontainerdevices.Rule{
-			{
-				Type:        'a',
-				Permissions: "rwm",
-				Allow:       true,
-				Minor:       libcontainerdevices.Wildcard,
-				Major:       libcontainerdevices.Wildcard,
-			},
-		},
 		SkipDevices: true,
```

Comment:
this intuitively makes sense to me, but could you please add a reference to the specs/docs (a line in the git commit is fine like `see: https://....`) for future reference?

Reply:
I was going to refer to https://github.com/opencontainers/runc/pull/2490, but unfortunately it falls short in case of systemd -- one more PR is needed (created: https://github.com/opencontainers/runc/pull/2958).

Removing this commit until https://github.com/opencontainers/runc/pull/2958 is merged, a release is cut, and kubernetes vendoring is updated to include it.